sql/core/src/test/resources/sql-tests/inputs/pgSQL/window.sql

```diff
@@ -0,0 +1,1169 @@
+        SELECT * FROM range(5, 49, 2)
+)
+SELECT x, (sum(x) over w)
+FROM cte
+WINDOW w AS (ORDER BY x range between 1 preceding and 1 following);
+
+WITH cte (x) AS (
+        select 1 union all select 1 union all select 1 union all
+        SELECT * FROM range(5, 49, 2)
+)
+SELECT x, (sum(x) over w)
+FROM cte
+WINDOW w AS (ORDER BY x range between 1 preceding and 1 following);
+
+-- with UNION
+
+SELECT count(*) OVER (PARTITION BY four) FROM (SELECT * FROM tenk1 UNION ALL SELECT * FROM tenk2)s LIMIT 0;
+
+-- check some degenerate cases
+
+create table t1 (f1 int, f2 int) using parquet;
+insert into t1 values (1,1),(1,2),(2,2);
+
+select f1, sum(f1) over (partition by f1 order by f1
+                         range between 1 preceding and 1 following)
+from t1 where f1 = f2; -- error, must have order by
+
+-- Since EXPLAIN clause rely on host physical location, it is commented out
+-- explain
+-- select f1, sum(f1) over (partition by f1 order by f2
+-- range between 1 preceding and 1 following)
+-- from t1 where f1 = f2;
+-- select f1, sum(f1) over (partition by f1 order by f2
+--                          range between 1 preceding and 1 following)
+-- from t1 where f1 = f2;
+
+select f1, sum(f1) over (partition by f1, f1 order by f2
+range between 2 preceding and 1 preceding)
+from t1 where f1 = f2;
+
+select f1, sum(f1) over (partition by f1, f2 order by f2
+range between 1 following and 2 following)
+from t1 where f1 = f2;
+
+select f1, sum(f1) over (partition by f1,
+f1 order by f2 range between 1 preceding and 1 following)
+from t1 where f1 = f2;
+
+-- Since EXPLAIN clause rely on host physical location, it is commented out
+-- explain
+-- select f1, sum(f1) over (partition by f1 order by f2
+-- range between 1 preceding and 1 following)
+-- from t1 where f1 = f2;
+
+select f1, sum(f1) over (partition by f1 order by f2
+range between 1 preceding and 1 following)
+from t1 where f1 = f2;
+
+select f1, sum(f1) over (partition by f1, f1 order by f2
+range between 2 preceding and 1 preceding)
+from t1 where f1 = f2;
+ 
+select f1, sum(f1) over (partition by f1, f2 order by f2
+range between 1 following and 2 following)
+from t1 where f1 = f2;
+
+-- ordering by a non-integer constant is allowed
+
+SELECT rank() OVER (ORDER BY length('abc'));
+
+-- can't order by another window function
+SELECT rank() OVER (ORDER BY rank() OVER (ORDER BY random()));
+
+-- some other errors
+select * from
+(select row_number() over (order by salary) rn from empsalary) ss
+where rn < 10;
+
+-- [SPARK-28506] not handling usage of group function and window function at some conditions
+-- SELECT * FROM empsalary INNER JOIN tenk1 ON row_number() OVER (ORDER BY salary) < 10;
+
+-- [SPARK-28506] not handling usage of group function and window function at some conditions
+-- SELECT rank() OVER (ORDER BY 1), count(*) FROM empsalary GROUP BY 1;
+
+-- [SPARK-28086] Adds `random()` to Spark
+-- SELECT * FROM rank() OVER (ORDER BY random());
```

Comment:
Remove line 950 and uncomment 951 since SPARK-28086 is merged.

Reply:
Oh. Please file another JIRA and keep this as the commented status. 
```
spark-sql> SELECT * FROM rank() OVER (ORDER BY random());
Error in query:
mismatched input 'BY' expecting {')', ',', '-'}(line 1, pos 33)

== SQL ==
SELECT * FROM rank() OVER (ORDER BY random())
---------------------------------^^^
```

Reply:
Hmm, I just checked out and it also does not work for PgSQL:
```sql
SELECT * FROM rank() OVER (ORDER BY random());
ERROR:  syntax error at or near "ORDER"
LINE 1: SELECT * FROM rank() OVER (ORDER BY random());
```
Although the error message is different, an error being thrown looks like the correct behaviour. What do you think? Should I JIRA about the error message being different or should I JIRA a fix to the error itself?

Reply:
Oh. I see. Then, there is no need to file a JIRA. Just uncomment this since we need to verify this failure behavior.

Reply:
Sure. By the way, the CI will not pass then, I think. There's a way to disable the error checking for this specific query?

Reply:
Ah, got it. It's due to `random()` seed issue. Then, please add a comment for that `random seed` issue.

Reply:
Done.